recipe/meta.yaml

```diff
@@ -47,11 +47,16 @@ test:
     - jupyter kernelgateway --help
 
 about:
-  home: http://github.com/jupyter/kernel_gateway
-  license: BSD 3-Clause
+  home: https://github.com/jupyter/kernel_gateway
+  license: BSD-3-Clause
   license_file: LICENSE.md
+  license_family: BSD
+  license_url: https://opensource.org/licenses/BSD-3-Clause
```

Comment:
LGTM.

nit: I think we are being discouraged from putting `license_url` and `doc_source_url` when `license_file` and `doc_url` are present (even though the linter barks). 

If you want to silence the linter you can use: 
```
extra:
  skip-lints:
    - missing_license_url
    - missing_doc_source_url

```

Reply:
Thanks Eric! I was unaware of this! I’ll add it to the meta.yaml